sdk/cognitivelanguage/azure-ai-language-conversations/setup.py

```diff
@@ -19,7 +18,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.16.0"]
+REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.19.0"]
```

Comment:
the setup.py should be updated to look like [this](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/setup.py). This is more of a generated basic setup.py, but we want to add things like the readme to the package